docusaurus/video/docusaurus/docs/api/recording/recording_calls.mdx

```diff
@@ -96,7 +96,7 @@ An error will be returned if the recording doesn't exist.
 <TabItem value="js" label="JavaScript">
 
 ```js
-call.deleteRecording(sessionID, filename);
+call.deleteRecording({ session: '<session id>', filename: '<filename>' })
```

Comment:
just out of curiosity, why using an object?

Reply:
Since most of the node-sdk is autogenerated, we don't need to manually update the method signature in case a new option is added to the request object. It's unlikely this will happen with this method, but since all other methods in the SDK follow this pattern, it makes sense to follow it for all methods.